lib/stack.dart

```diff
@@ -17,6 +17,9 @@ class Stack<T> {
     _sizeMax = noLimit;
   }
 
+  // Returns a list of T elements contained in the Stack
```

Comment:
Can you change the `//` to `///` 
https://dart.dev/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types